test-cases/private-repo/private-repo.yaml

```diff
@@ -0,0 +1,9 @@
+cluster_backend: 'gce'
+logs_transport: 'ssh'
+n_db_nodes: 0
+n_loaders: 0
+n_monitor_nodes: 0
+store_results_in_elasticsearch: False
+nemesis_class_name: 'NoOpMonkey'
+user_prefix: 'private-repo'
+scylla_repo: 'https://repositories.scylladb.com/scylla/repo/amos-test-centos/centos/scylladb-1.7.repo'
```

Comment:
Let's rename the UUID `amos-test-centos` to `qa-test-centos`

Reply:
done